tests/exhaustive/autogen/gen02/expected/model_1_1.eprime

```diff
@@ -35,8 +32,8 @@ such that
              | q3 : int(1..4)]),
     and([q4 <= var2_ExplicitVarSizeWithMarkerR5_Marker ->
          (2 <= var2_ExplicitVarSizeWithMarkerR5_Values_ExplicitVarSizeWithMarker_Marker[q4] ->
-          -toInt(var2_ExplicitVarSizeWithMarkerR5_Values_ExplicitVarSizeWithMarker_Values[q4, 1]) <
-          -toInt(var2_ExplicitVarSizeWithMarkerR5_Values_ExplicitVarSizeWithMarker_Values[q4, 2]))
+          [-toInt(var2_ExplicitVarSizeWithMarkerR5_Values_ExplicitVarSizeWithMarker_Values[q4, 1]); int(1)] <lex
```

Comment:
I suggest we rewrite [x] <lex [y] (and <=lex) to < (and <=). This won't make a difference inside of SR really, but let's do the simplification since it is easy for us to do. What do you say?

Reply:
I assume savilerow does the same for <lex and < on 1D matrix of Int. That should be fine! Will mean fewer conflicts with master :)

Reply:
Okay, I see what you mean. I've added a post processing rule to find AbsLitMatrix's containing a single value either side of a lex and rewrite them to regular less thans.